performance/libs/create-author1-data.js

```diff
@@ -0,0 +1,40 @@
+const md5 = require("md5");
+const { createApolloFetch } = require("apollo-fetch");
+
+async function createAuthor1() {
```

Comment:
> > meaning that one can only launch the `yarn compare` command and see the results without having to run the benchmarks. Thoughts?
> 
> Sounds nice, what would `yarn compare` do, open up a single view table with the different results?

So compare will just do a heuristic comparison between two server versions and tell if one server is fast than the other, or they are similar. To have a detailed view of the statistics, once can use `yarn compare:stats`

Reply:
> Are there concurrent mutations and subscriptions taking place or everything happening asychronously?

Current, this script is only used to seed data to a running instance. No subscriptions are being benchmarked, them being active by default, I do not think they will interfere here (since create/delete/update) are not being tested.